br/tests/lightning_fail_fast_on_nonretry_err/run.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -eu
+
+run_sql 'DROP DATABASE IF EXISTS csv'
+
+out_file_name="$TEST_DIR/sql_res.$TEST_NAME.txt"
+export GO_FAILPOINTS="github.com/pingcap/tidb/br/pkg/lightning/backend/local/WriteToTiKVNotEnoughDiskSpace=return(true)"
+start_time=$(date +'%s')
+run_lightning --backend local > $out_file_name 2>&1 || true
```

Comment:
I think the `|| true` is not necessary.  Instead we can check if the return code is 0, it is not expected and exit early.

Reply:
check https://github.com/pingcap/tidb/pull/33794#discussion_r849020070